apps/io-services-cms-webapp/src/utils/apim-proxy.ts

```diff
@@ -0,0 +1,49 @@
+import { ApiManagementClient } from "@azure/arm-apimanagement";
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import * as TE from "fp-ts/lib/TaskEither";
+import { pipe } from "fp-ts/lib/function";
+import {
+  ApimRestError,
+  getSubscription,
+  getUser,
+  getUserGroups,
+  parseOwnerIdFullPath,
+} from "../apim_client";
+import { Delegate } from "./service_review_proxy";
+
+export const apimProxy = (
+  apimClient: ApiManagementClient,
+  apimResourceGroup: string,
+  apim: string
+) => ({
+  getDelegateFromServiceId: (
```

Comment:
Please add a doc block to explain what this methods do

Reply:
[4c732a9](https://github.com/pagopa/io-services-cms/pull/25/commits/4c732a9a92b60bb0a847a99868385359dbbd247c)